content/TrueCommand/TCGettingStarted/Install/InstallTCCloud.md

```diff
@@ -0,0 +1,195 @@
+---
+{{< toc >}}
+
+TrueCommand is versatile and offers several different install options.
+TrueCommand Cloud is the preferred method for using TrueCommand since it requires no local resources or specific hardware requirements to get started!
+
+{{< hint info >}}
+A credit card number is required for Cloud Signups.
+
+If the TrueCommand Cloud license was purchased as part of an Enterprise contract, please contact [**iXSystems Customer Support**]({{< relref "/TrueCommand/Introduction/Support.md" >}}) and a member of the Deployment Team will provide a sign-up code which makes the cost $0 for the duration of the contract.
+{{< /hint >}}
+
+## Begin TrueCommand Cloud Installation
+
+{{< expand "TrueCommand Cloud: Register and Create Subscription" >}}
+
+TrueCommand Cloud is a SaaS offering of TrueCommand with a WireGuard VPN capability to connect TrueNAS systems through firewalls.
+TrueCommand Cloud is compatible with TrueNAS version v12.0 and newer.
+
+### Register an iXsystems Account
+
+Open https://portal.ixsystems.com and click **Register**. 
+
+![PortalLogin](/images/TrueCommand/Cloud/PortalLogin.png "Portal Login")
+
+Fill out the form using the email address you want to use. 
+
+![PortalAccountCreate](/images/TrueCommand/Cloud/PortalAccountCreate.png "Portal Account Create")
+
+Check the address spam folder if the email does not arrive within a few minutes.
+If the email is in the spam folder, mark it as *not spam* and add the account to the address book.
+After receiving the verification email, open the link provided to verify the account.
+
+### Create a New Subscription
+
+Log in to the verified account and click **New Subscription**.  
+
+![PortalDashboard](/images/TrueCommand/Cloud/PortalDashboard.png "Account Services: Dashboard")
+
+![PortalCloudSubscription](/images/TrueCommand/Cloud/PortalCloudSubscription.png "Creating a Cloud Subscription")
+
+Select the TrueCommand Cloud option and choose the subscription plan that best fits your current needs.
+
+![PortalCloudSubscriptionOptions](/images/TrueCommand/Cloud/PortalCloudSubscriptionOptions.png "Account Services: Cloud Subscription Options")
+
+Click **Continue** to proceed.
+
+![PortalCloudSubscriptionSelected](/images/TrueCommand/Cloud/PortalCloudSubscriptionSelected.png "Account Services: Cloud Subscription Selected")
+
+Next, fill out the payment form. 
+
+![CloudSubscriptionPayment](/images/TrueCommand/Cloud/CloudSubscriptionPayment.png "Cloud Subscription Payment")
+
+Click Subscribe and wait for the form to be accepted. The Account Services window confirms that you have an active TrueNAS Cloud account with an active subscription.
+
+### Provision the Active Subscription
+
+Click the **Provision Now** button which is located to the right of the Active Subscription listing.
+
+![PortalDashboardActiveSubscription](/images/TrueCommand/Cloud/PortalDashboardActiveSubscription.png "Account Services: Active Subscriptions")
+
+Select a **Subnet** that your network is not using.
```

Comment:
in this case subnet is the value for the field of the same name, but we should avoid combining the value with the field name, so try this:
```suggestion
Select a subnet that your network is not using from the **Subnet** dropdown list.
```

Reply:
Made suggested changes.